CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+# Guidelines for contributing to the Referencing discussion at the JSON Schema Org
+
+## Issues
+
+Please use issues to either discuss general principels or give feedback on specific proposals.
+
+## Pull requests
+
+Please use pull requests from a repository fork to submit new proposals.  PRs for minor fixes (spelling, formatting) are welcome at any time.  PRs for larger changes should be discussed in issues first.
+
+During this intial phase, proposal authors can determine when to merge PRs against their proposals.  This will change once we settle on a proposal to take into the standardization process.
+
+## Conduct
+
+All official channels including the mailing list, GitHub organization, and Freenode channel, follow the IETF Guidelines for Conduct as specified in [RFC7154](https://tools.ietf.org/html/rfc7154).
```

Comment:
This should defer to our org wide Code of Conduct which includes the IETF BCP 54 "IETF Guidelines for Conduct" (RFC7154) by way of reference.